run_cart_sockets.sh

```diff
@@ -1,35 +1,35 @@
 #!/bin/bash
 #---------------
-#Run CaRT tests 
+#Run CaRT tests
 #---------------
 
 #SBATCH -J test_daos1           # Job name
 #SBATCH -o stdout.o%j           # Name of stdout output file
 #SBATCH -e stderr.e%j           # Name of stderr error file
 #SBATCH -A STAR-Intel           # Project Name
-#SBATCH -p normal           # Queue (partition) name
+#SBATCH -p development          # Queue (partition) name
 #SBATCH -N 10                   # Total # of nodes
-#SBATCH -n 100                  # Total # of mpi tasks (48 x  Total # of nodes)
-#SBATCH -t 1:00:00              # Run time (hh:mm:ss)
+#SBATCH -n 10                   # Total # of mpi tasks (48 x  Total # of nodes)
+#SBATCH -t 0:30:00              # Run time (hh:mm:ss)
 #SBATCH --mail-user=first.last@intel.com
 #SBATCH --mail-type=all         # Send email at begin and end of job
 
-
 #Parameter to be updated for each sbatch
 N_SERVERS=2
 N_CLIENTS=1
 M_SERVERS=8
 MAX_INFLIGHT=16
 CARTDIR="/home1/<PATH_TO_CART>/cart"
 
-interface=ib0
+interface=em1
```

Comment:
Specific reason ib0 was removed? If I understand right, we can run sockets with ib0 and have a bit quicker run isn't it?

Reply:
yep, good catch. will update in patch2.